acq4/drivers/pvcam/pvcam.py

```diff
@@ -60,9 +60,16 @@
         ('binningX', 1, [1,2,3,4,8,12,16,24,32,48,64,128,256,512]),
         ('binningY', 1, [1,2,3,4,8,12,16,24,32,48,64,128,256,512]),
     ],
+
+    'RetigaR1': [
+        ('READOUT_PORT', 0),  ## Fastest port for R1
+        ('SPDTAB_INDEX', 2),  ## Fastest option for Q57
```

Comment:
I've poked around a bit - the R1 is pretty standard, and does have 2 ports (14 and 16 bit), and 3 speeds (the cameras come with a certfication for the readout noise for all 6 combinations of settings). Obviously the comment should no longer say Q57. I haven't found any documentation that precisely says how the port values are mapped to the speed or the readout, however.